save-backend/src/main/kotlin/com/saveourtool/save/backend/controllers/vulnerability/VulnerabilityController.kt

```diff
@@ -253,9 +256,12 @@ class VulnerabilityController(
     fun saveDate(
         @RequestBody vulnerabilityDateDto: VulnerabilityDateDto,
         authentication: Authentication,
-    ): Mono<StringResponse> = blockingToMono {
-        vulnerabilityService.saveDate(vulnerabilityDateDto, authentication)
-    }
+    ): Mono<StringResponse> = vulnerabilityDateDto.vulnerabilityName.toMono()
+        .filter {
+            vulnerabilityPermissionEvaluator.hasPermission(authentication, it, Permission.DELETE)
```

Comment:
I thought you will have permissions like user/admin/etc., but you have it as attributes: Delete, Edit, etc.
Don't know which is the best one

Reply:
As we already have several `PermissionEvaluator`s, I would support existing attribute system